lib/languages/js/templates/api.js.tpl

```diff
@@ -75,6 +75,7 @@ module.exports = function (options) {
             if (key !== 'message') { err[key] = errorData[key]; }
           });
           err.statusCode = axiosError.response.status;
+          if (err.statusCode === 413) { err.message = 'Request entity too large.'}
```

Comment:
lets only do this if there is no message

Reply:
what is the current error message for 413? I thought it said something about a network error, which is misleading IMO.